src/main/java/nextstep/subway/line/ui/LineController.java

```diff
@@ -25,4 +31,29 @@ public ResponseEntity createLine(@RequestBody LineRequest lineRequest) {
         LineResponse line = lineService.saveLine(lineRequest);
         return ResponseEntity.created(URI.create("/lines/" + line.getId())).body(line);
     }
+
+    @PutMapping("/{id}")
+    public ResponseEntity updateLine(@RequestBody LineRequest lineRequest,
+        @PathVariable("id") Long id) {
+        LineResponse line = lineService.update(lineRequest, id);
+        return ResponseEntity.ok().body(line);
+    }
+
+    @GetMapping
+    public ResponseEntity getLines() {
+        List<LineResponse> lines = lineService.getLines();
+        return ResponseEntity.ok().body(lines);
+    }
+
+    @GetMapping("/{id}")
+    public ResponseEntity getLine(@PathVariable("id") Long id) {
+        LineResponse line = lineService.getLine(id);
+        return ResponseEntity.ok().body(line);
+    }
+
+    @DeleteMapping("/{id}")
+    public ResponseEntity delete(@PathVariable("id") Long id) {
+        lineService.delete(id);
+        return ResponseEntity.ok().body(null);
```

Comment:
```suggestion
        return ResponseEntity.noContent().build();
```

요구사항에 삭제시 응답 코드는 204입니다.

Reply:
해당부분 확인하고 noContent() 코드를 적용하였습니다!